### ISSUE TITLE
feat: review/dismiss events on the v0.14 dashboard

### ISSUE THREAD

**Issue by tedder** [blakeblackshear/frigate#12911]
**Describe what you are trying to accomplish and why in non technical terms**

I want to have a way to clear the new events right on the dashboard.

**Describe the solution you'd like**

Currently the dashboard simply shows the cameras unless there are new events. The new events are in a row at the top. I can click to go to `/review` and review it, but there's no way to simply clear all events while staying on the dashboard.

Since the dashboard is the default view, and it's handy to use as a live feed, the activity on events is distracting.

Basically I'm looking for a "mark all as reviewed" or "archive all" button.

**Comment by hawkeye217** [comment]
[This PR](https://github.com/blakeblackshear/frigate/pull/12878) was recently committed to add a small button to the alerts in the filmstrip to be able to quickly dismiss them. It's not a "mark all as reviewed", but it does give a quick way to mark them as reviewed more quickly.

This is currently available in the dev builds and will be available in the next maintenance release of Frigate.
